app/processing/get-completed-payment-requests.js

```diff
@@ -1,13 +1,12 @@
 const db = require('../data')
+const { getCompletedPaymentRequestsFilter } = require('./get-completed-payment-requests-filter')
 
-const getCompletedPaymentRequests = async (schemeId, frn, marketingYear, agreementNumber, paymentRequestNumber) => {
+const getCompletedPaymentRequests = async (paymentRequest) => {
+  const filter = getCompletedPaymentRequestsFilter(paymentRequest)
   const completedPaymentRequests = await db.completedPaymentRequest.findAll({
     where: {
-      schemeId,
-      frn,
-      marketingYear,
-      agreementNumber,
-      paymentRequestNumber: { [db.Sequelize.Op.lt]: paymentRequestNumber },
+      ...filter,
+      paymentRequestNumber: { [db.Sequelize.Op.lt]: paymentRequest.paymentRequestNumber },
```

Comment:
Same again here, if you could just summarise what this does that would be great :)

Reply:
Same as before, it's a helper to translate into SQL code.  In this case `db.Sequelize.Op.lt` equals "less than".

So this would translate as:

`WHERE "paymentRequestNumber" < 3` (if current payment request number was 3)